source/tests/system/nirfmxcdma2k_driver_api_tests.cpp

```diff
@@ -64,6 +72,334 @@ TEST_F(NiRFmxCDMA2kDriverApiTests, Init_Close_Succeeds)
   EXPECT_RESPONSE_SUCCESS(close_response);
 }
 
+TEST_F(NiRFmxCDMA2kDriverApiTests, AcpFromExample_FetchData_DataLooksReasonable)
+{
+  const auto NUMBER_OF_OFFSETS = 2;
+  int32 autoLevel = 1;
+  // TODO: Conversion process ignored auto_level_response.reference_level
```

Comment:
Is this something you're intending to fix with this PR or is there a separate work item to circle back on this?

Reply:
No action is needed, here I would like to remove this comment. For it is generated by the ex_to_test.py script.